index.html

```diff
@@ -17,46 +23,48 @@
                 // hint: these are strings
                 var numberOne = document.forms.inputs.numberOne.value;
                 var numberTwo = document.forms.inputs.numberTwo.value;
-                document.forms.inputs.result.value = numberOne + numberTwo;
+                document.forms.inputs.result.value = parseInt(numberOne) + parseInt(numberTwo);
             },
             subtractNumbers: function () {
-                
+                var numberOne = document.forms.inputs.numberOne.value;
+                var numberTwo = document.forms.inputs.numberTwo.value;
+                document.forms.inputs.result.value = parseInt(numberOne) - parseInt(numberTwo);
             }
         }
 
     </script>
 
 </head>
 
-<body>
-    <div>Header</div>
+<body style="background-color: rgb(148, 109,221);">
+    <h1 style="color: rgb(114, 12, 12); text-align: center;">Enter your numbers into the fields below and hit the corresponding button for addition or subtraction!</h1>
```

Comment:
Consider moving this style information into the style tag with an appropriate css selector / class name.